vividus-allure-adaptor/src/main/java/org/vividus/report/allure/plugin/CustomTabPlugin.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019-2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.report.allure.plugin;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang3.StringUtils;
+
+import io.qameta.allure.PluginConfiguration;
+import io.qameta.allure.plugin.DefaultPlugin;
+
+public final class CustomTabPlugin extends DefaultPlugin
+{
+    private static final String INDEX_JS = "index.js";
+    private static final String PLUGIN_ID = "custom-tab";
+    private static final String FILES_FOLDER_PATH = "/plugin/" + PLUGIN_ID;
```

Comment:
```suggestion
    private static final String FILES_FOLDER_PATH = "/report-plugin/" + PLUGIN_ID;
```
let's use `report-plugin` as a base dir for user data